apps/src/templates/sectionAssessments/CorrectAnswerIcon.jsx

```diff
@@ -1,19 +1,54 @@
-import React, {Component} from 'react';
+import React, {Component, PropTypes} from 'react';
 import color from "@cdo/apps/util/color";
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 
 const styles = {
+  main: {
+    border: 'none',
+    height: '30px',
+    marginTop: '20px',
+    marginRight: '80px',
+    paddingTop: '20px',
+    paddingLeft: '20px',
+    paddingRight: '10px',
+    paddingBottom: '10px',
+  },
   icon: {
     color: color.level_perfect,
-  }
+    float: 'left',
+    width: '10%',
+    fontSize: '25px',
+  },
+  text: {
+    color: color.black,
+    fontWeight: 'bold',
+    fontFamily: 'Gotham 3r',
+    fontSize: '14px',
+    float: 'left',
+    paddingRight: '40px',
+    width: '10%',
+  },
 };
 
-export default class CorrectAnswerIcon extends Component {
+class CorrectAnswerIcon extends Component {
+  static propTypes = {
+    percentValue: PropTypes.string.isRequired,
+    isCorrectAnswer: PropTypes.bool.isRequired,
+  };
+
   render() {
+    const {percentValue} = this.props;
     return (
-      <div>
-        <FontAwesome icon="check-circle" style={styles.icon}/>
+      <div style={styles.main}>
+        <div style={styles.text}>
+          {percentValue}
+        </div>
+        <div>
```

Comment:
You did a nice job defining `isCorrectAnswer` as a boolean propType, but you didn't actually use it to hide/show the checkmark. Try something like:

```
{this.props.isCorrectAnswer &&
  <FontAwesome icon="check-circle" style={styles.icon}/>
}
```
This will hide and how the element based your property that gets passed in.

Reply:
Fixed.